assets/config_identity_suite.json

```diff
@@ -5,6 +5,7 @@
   },
   "sdk": {
     "baseUrl": "https://mycompany.com",
+    "facetId": "android:apk-key-hash:ch.nevis.mobile.authentication.sdk.flutter.example",
```

Comment:
Shouldn't we add this to cloud config as well?

Reply:
You're right, I left that file out from the commit. Thanks!